lib/ui/collections/remote_collections_grid_view_widget.dart

```diff
@@ -14,6 +14,7 @@ class RemoteCollectionsGridViewWidget extends StatelessWidget {
   static const maxThumbnailWidth = 224.0;
   static const fixedGapBetweenAlbum = 8.0;
   static const minGapForHorizontalPadding = 8.0;
+  static const collectionItemsToPreload = 100;
```

Comment:
why keep such high value? 10 or 15 should be more than enough, no?

Reply:
Reduced it to 20.